assistant/src/main/java/com/ibm/watson/assistant/v1/model/Counterexample.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 IBM Corp. All Rights Reserved.
+ * (C) Copyright IBM Corp. 2019.
```

Comment:
Did all copyright years change to 2019? We might want to look into that

Reply:
Just pushed a commit that changed all the remaining ones.